Source/SwiftLintBuiltInRules/Rules/Style/SortedEnumCasesRule.swift

```diff
@@ -91,7 +110,7 @@ private extension SortedEnumCasesRule {
 
             let cases = node.memberBlock.members.compactMap { $0.decl.as(EnumCaseDeclSyntax.self) }
             let sortedCases = cases
-                .sorted(by: { $0.elements.first!.name.text < $1.elements.first!.name.text })
```

Comment:
I recommend using `caseInsensitiveCompare` to have a clearer intention.

Reply:
Good suggestion. This is also likely more efficient as that function can return early as opposed to `lowercase` bringing both string to the same case regardless of their content 👍